java/automl/automl-common/pom.xml

```diff
@@ -10,6 +10,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>automl-common</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.github.qingguox</groupId>
+            <artifactId>Crescent-Toolkit</artifactId>
+            <version>${crescent-utls}</version>
```

Comment:
There is no need to specify the version. The version of the dependencies in project is managed in the parent pom.

Reply:
ok, utils-version has bean modified. see new commit from https://github.com/ml-for-good/automl/pull/110/commits/a3bb6a630a911f5a133c15e742ff388b422194f7 .